docs/sources/panels-visualizations/visualizations/geomap/index.md

```diff
@@ -38,9 +38,9 @@ weight: 100
 
 # Geomap
 
-The Geomap panel visualization allows you to view and customize the world map using geospatial data. You can configure various overlay styles and map view settings to easily focus on the important location-based characteristics of the data.
+Geomaps allow you to view and customize the world map using geospatial data. You can configure various overlay styles and map view settings to easily focus on the important location-based characteristics of the data.
 
-> We would love your feedback on Geomap. Please check out the [Github discussion](https://github.com/grafana/grafana/discussions/62159) and join the conversation.
+> We would love your feedback on geomaps. Please check out the [Github discussion](https://github.com/grafana/grafana/discussions/62159) and join the conversation.
```

Comment:
Can we keep this singular?

Reply:
Just in this particular instance or in general? And why?

Reply:
This does sound a little funny but I think we can get used to the plural version 🤷‍♂️

Reply:
I suspect this one just may be that we're accustomed to treating it like a brand name so it's a shift to stop doing that. But grammatically, it's solid, so I think we should stick with the our new convention here.